### PR TITLE
fix: changing the title and body content in quick succession would trigger unsaved changes warning prompt

### LINKED ISSUE #2949: "unsaved changes" alert upon document creation
Upon the creation of a document, if the title is set, followed by any operations in the content area prior to the initial save, an error alert with "you have unsaved changes" pops up when the initial save is triggered. Regardless of which option is chosen, no changes to the new document are made. Additionally, if the document is manually saved as a draft or published prior to the initial save, the issue still occurs.

**To Reproduce**
1. Create a new document.
2. Add a title and add content within ~5 seconds of the document creation.
3. See error.

**Expected behavior**
Document should save without the alert.

**Screenshots**
Screen Shot 2022-01-18 at 8 43 02 AM

**Outline (please complete the following information):**
- Install: self hosted
- Version: d0e7f2de65b63f94bdccb99e0d5234af3426799f

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome
 - Version 96.0.4664.110

### PR DESCRIPTION
**Pull request by thenanyu**
fixes #2949

### CODE REVIEW

File: shared/i18n/locales/en_US/translation.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
   "Dark": "Dark",
   "Light": "Light",
   "System": "System",
+  "Appearance": "Appearance",
```
Comment (thenanyu) [inline shared/i18n/locales/en_US/translation.json:43]: I have no idea what this is lol

Comment (tommoor) [reply shared/i18n/locales/en_US/translation.json:43]: auto-generated – it should be on master already but nbd

File: app/scenes/Document/components/Document.tsx [hunk 2]
```diff
@@ -466,9 +466,20 @@ class DocumentScene extends React.Component<Props> {
...
                   }
-                  message={t(
-                    `You have unsaved changes.\nAre you sure you want to discard them?`
-                  )}
+                  message={(location, action) => {
+                    if (
+                      // a URL replace matching the current document indicates a title change
+                      // no guard is needed for this transition
+                      action === "REPLACE" &&
+                      location.pathname.includes(document.url)
```
Comment (tommoor) [inline app/scenes/Document/components/Document.tsx]: Can we use just the edit path?

**Comment by thenanyu** [comment]
@tommoor I think this should be good now, just tested

**Review by tommoor — APPROVED** [review]

**Pull request merged**